cli/azd/pkg/graphsdk/graphsdk.go

```diff
@@ -0,0 +1,10 @@
+package graphsdk
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+)
+
+var ServiceConfig cloud.ServiceConfiguration = cloud.ServiceConfiguration{
```

Comment:
Not for now, but once we've completed the work to stop using the `az` CLI, we probably need to do some thinking about testing in different clouds instead of just the public cloud.  I know some of the SDK teams (Identity, IIRC?) have ways to run their tests against different clouds.

Reply:
Yeah, this is a setting that you can pass in the `ClientOptions` object.  Agreed, but not right now.

Reply:
Added issue #973 to track this.